AlfrescoApp/View Controllers/Accounts View Controllers/New Account/NewAccountViewController.m

```diff
@@ -441,7 +441,7 @@ - (void)constructTableCellsForAlfrescoServer
         protocolCell.titleLabel.text = NSLocalizedString(@"accountdetails.fields.protocol", @"HTTPS protocol");
         self.protocolSwitch = protocolCell.valueSwitch;
         [self.protocolSwitch addTarget:self action:@selector(protocolChanged:) forControlEvents:UIControlEventValueChanged];
-        BOOL isHTTPSOn = self.formBackupAccount.protocol ? [self.formBackupAccount.protocol isEqualToString:kProtocolHTTPS] : NO;
+        BOOL isHTTPSOn = self.formBackupAccount.protocol ? [self.formBackupAccount.protocol isEqualToString:kProtocolHTTPS] : YES;
```

Comment:
`AccountInfoDetailsViewController.m` has the original version of this line; what's different between that class and this one that means the logic needs to be different?